spec/std/io/multi_writer_spec.cr

```diff
@@ -49,4 +49,19 @@ describe "IO::MultiWriter" do
       io.closed?.should eq(true)
     end
   end
+
+  describe "#flush" do
+    it "writes to IO and File" do
+      file = File.tempfile("foo")
```

Comment:
Sry, I meant the `with_tempfile` helper from `spec/support/tempfile.cr`. It automatically cleans up the tempfile after the spec is run.

Reply:
done